### PR TITLE
Added WG Specification Meeting Summary

### PR DESCRIPTION
**Pull request by brad1121**

### CODE REVIEW

**Comment by schancel** [comment]
I'll merge this in a few days if there are no other comments.

**Comment by brad1121** [comment]
This now includes the OP GROUP meeting summary as well.

**Comment by schancel** [comment]
Can you please submit the other meeting summary as a different pull request?  I would like to merge the one that has been up for a week since it received no feedback -- but the new one needs some time for comments.

File: wg-opcodes/summaries/20180214 - Meeting Summary.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+     * Requires more changes which need to be sold with the implementation
+     * Requires wallet support
+         * Cannot be tested without a wallet - BU has a full node + wallet available.
+             * https://github.com/gandrewstone/BitcoinUnlimited/tree/opgroup_fullnode
+             * https://github.com/gandrewstone/BitcoinUnlimited/tree/opgroup_consensus 
+         * 3 Months till Fork Date and 2 months to be fully tested
+     * Discussion about other aspects around changing the transaction version
+         * Is it limited to OP_GROUP
+         * Does it extend to other bugs than could be done with a TX version
+         * Do we bump the TX version with the reenabling of old OP_CODES
```
Comment (dgenr8) [inline wg-opcodes/summaries/20180214 - Meeting Summary.md]: I should think not.
 - The transaction format has not changed
 - Re-enabling is a hard fork so wallets have to upgrade anyway

**Comment by dgenr8** [comment]
> Do we bump the TX version with the reenabling of old OP_CODES

I should think not:
 - The transaction format has not changed
 - Re-enabling is a hard fork so wallets have to upgrade anyway

**Comment by brad1121** [comment]
updated this to only include wg-protocol-spec notes as requested

**Comment by schancel** [comment]
@dgenr8

I agree with your assessment there.  This seems to be a reply to what happened in the meeting rather than a comment about anything that needs to be fixed in the document.  As such, I'll leave it here and merge as-is.

**Pull request merged**